mongodbatlas/data_source_mongodbatlas_privatelink_endpoint_service_adl.go

```diff
@@ -33,6 +33,7 @@ func dataSourceMongoDBAtlasPrivateLinkEndpointServiceADL() *schema.Resource {
 				Computed: true,
```

Comment:
believe we are missing deprecation message on Data Source: privatelink_endpoint**s**_service_adl as well